resources/views/cart/overview.blade.php

```diff
@@ -5,93 +5,111 @@
 @section('robots', 'NOINDEX,NOFOLLOW')
 
 @section('content')
-    <h1 class="font-bold text-4xl mb-5">@lang('Cart')</h1>
-
-    <cart v-cloak>
-        <div v-if="hasItems" slot-scope="{ cart, hasItems, changeQty, remove }">
-            <div class="flex flex-wrap items-center border-b pb-2 mb-2" v-for="(item, itemId, index) in cart.items">
-                <div class="w-1/6 sm:w-1/12 pr-3">
-                    <a :href="item.url | url" class="block">
-                        <picture>
-                            <source :srcset="'/storage/{{ config('rapidez.store') }}/resizes/80x80/magento/catalog/product' + item.image + '.webp'" type="image/webp">
-                            <img
-                                :alt="item.name"
-                                :src="'/storage/{{ config('rapidez.store') }}/resizes/80x80/magento/catalog/product' + item.image"
-                                height="100"
-                                class="mx-auto"
-                            />
-                        </picture>
-                    </a>
-                </div>
-                <div class="w-5/6 sm:w-5/12 lg:w-8/12">
-                    <a :href="item.url | url" dusk="cart-item-name" class="font-bold">@{{ item.name }}</a>
-                    <div v-for="(optionValue, option) in item.options">
-                        @{{ option }}: @{{ optionValue }}
+    <div class="container">
+        <h1 class="mb-5 text-4xl font-bold">@lang('Cart')</h1>
+        <cart v-cloak>
+            <div v-if="hasItems" slot-scope="{ cart, hasItems, changeQty, remove }">
+                <div class="flex flex-col gap-x-6 lg:flex-row">
+                    <div class="flex w-full flex-col">
+                        <div class="mb-4 flex flex-wrap items-center border-b pb-2" v-for="(item, itemId, index) in cart.items">
+                            <div class="w-1/6 pr-3 sm:w-1/12">
+                                <a class="block":href="item.url | url">
+                                    <img
+                                        class="mx-auto"
+                                        :alt="item.name"
+                                        :src="'/storage/{{ config('rapidez.store') }}/resizes/80x80/magento/catalog/product' + item.image + '.webp'"
+                                        height="100"
+                                    />
+                                </a>
+                            </div>
+                            <div class="w-5/6 sm:w-5/12 xl:w-6/12">
+                                <a
```

Comment:
Maybe on 1 line like this: `<a :href="item.url | url" class="font-bold" dusk="cart-item-name">@{{ item.name }}</a>`